pkg/volume/volume.go

```diff
@@ -28,6 +28,22 @@ import (
 type Volume interface {
 	// GetPath returns the directory path the volume is mounted to.
 	GetPath() string
+
+	Accountable
+}
+
+type VolumeDefaults struct {
+	AccountingNil
```

Comment:
SGTM.  Is this an error condition right now, or should we consider accounting "best effort" and return a nil-error with an uninitialized struct for unsupported volumes?